test/levenshtein.lean

```diff
@@ -1,4 +1,5 @@
 import Mathlib.Data.List.EditDistance.Defs
+import Mathlib.Algebra.Group.Nat
```

Comment:
```suggestion
```

Reply:
Not sure why are you suggesting this, it fails without?

Reply:
Uh okay. I assumed it was a leftover from the state of the PR before #11924.